### PR TITLE
Sweep model-slices 64/128/256 (untested core capacity axis)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

`--model-slices` controls the number of latent "slice" attention tokens in the Transolver backbone — the core capacity parameter governing how many distinct flow patterns the model can attend over simultaneously. It has been set to 128 in **every single experiment on tay** and has never been varied. This makes it one of the highest-value untested axes in our search space.

For CFD surface tasks, the challenging fine-grained channels (tau_y and tau_z) have the largest gap vs AB-UPT (×3.27 and ×3.43 respectively). These components require the model to resolve subtle tangential shear gradients over the car body — a task that may benefit from more slice tokens (richer attention basis) or may actually improve with fewer, more focused tokens (better regularisation). The original Transolver paper explores 32–256 on ShapeNet and found significant sensitivity. We are flying blind at 128.

**Experiment design:** 3-arm DDP8 sequential sweep over slice counts bracketing the current SOTA value:
- **Arm A:** `--model-slices 64` (half capacity — test regularisation hypothesis)
- **Arm B:** `--model-slices 128` (control — reproduce SOTA on current stack to confirm no regression)
- **Arm C:** `--model-slices 256` (double capacity — test richer representation hypothesis)

All other hyperparameters locked to SOTA stack from PR #232.

## Instructions

Run 3 arms **sequentially** on your 8 GPUs (DDP8 per arm). Use `--wandb_group tay-round13-slices-sweep` for all arms so runs are grouped in W&B.

### Arm A — slices=64
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent fern --wandb_group tay-round13-slices-sweep \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 64 \
  --ema-decay 0.999 --lr-warmup-epochs 1
```

### Arm B — slices=128 (control)
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent fern --wandb_group tay-round13-slices-sweep \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.999 --lr-warmup-epochs 1
```

### Arm C — slices=256
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent fern --wandb_group tay-round13-slices-sweep \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 256 \
  --ema-decay 0.999 --lr-warmup-epochs 1
```

**After all 3 arms complete**, post a comment with a results table showing per-arm best-epoch val_abupt plus the per-axis breakdown (surface_pressure, volume_pressure, wall_shear, tau_x, tau_y, tau_z) and the W&B run IDs. Report test metrics from the best validation checkpoint of the best-performing arm.

## Baseline

**Current SOTA:** PR #232 (askeladd), W&B run `r8s2dtnq`, group `tay-round12-heads-4`

| Metric | Val (ep11 best) | Test | AB-UPT Target | Gap (×) |
|--------|----------------|------|---------------|---------|
| `abupt_axis_mean_rel_l2_pct` | **9.0650%** | 10.190% | — | — |
| `surface_pressure` | 5.703% | 5.461% | 3.82% | 1.43× |
| `volume_pressure` | 5.830% | 12.656% | 6.08% | **2.08×** |
| `wall_shear` | — | 9.910% | 7.29% | 1.36× |
| `tau_x` | — | 8.432% | 5.35% | 1.58× |
| `tau_y` | — | 11.952% | 3.65% | **3.27×** |
| `tau_z` | — | 12.447% | 3.63% | **3.43×** |

Beat target: `val_primary/abupt_axis_mean_rel_l2_pct < 9.0650%`

**Checkpoint selection metric:** `val_primary/abupt_axis_mean_rel_l2_pct` (lower is better)

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #318 because Closing before student starts — CURRENT_RESEARCH_STATE.md records 'model_slices | CLOSED NEGATIVE | 128 SOTA; 64 regression (PR #231)'. Arm A (slices=64) is a confirmed regression and Arm B (slices=128) is just the control. Only Arm C (slices=256) would be novel, and one arm doesn't warrant a full PR. Reassigning fern to a genuinely unexplored axis (U-net skip connections). Sorry for the noise — cross-checked state file too late..
